package.json

```
@@ -52,6 +52,7 @@
     ]
   },
   "dependencies": {
+    "aerospike": "^5.12.1",
```

please pin dependency
```suggestion
    "aerospike": "5.12.1",
```